doc/source/rllib/package_ref/policy.rst

```diff
@@ -0,0 +1,32 @@
+.. _policy-reference-docs:
+
+Policy APIs
+===========
+
+.. toctree::
+   :maxdepth: 1
+
+   policy/policy.rst
+   policy/tf_policies.rst
+   policy/torch_policy.rst
+   policy/custom_policies.rst
```

Comment:
unfortunately, after using this page a bit, I've realized that this is a bit hard to realize that it is linking to actual API references. Maybe you can move this toctree to the bottom of the page with a section called 'API Reference'?

Reply:
done